diff --git a/dist/module.mjs b/dist/module.mjs
index c0737316145fdb26066befb92fbd2bb5af0cdf56..d6b927b9728d7e4a1694695fe88cce6995a8117c 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -217,7 +217,7 @@ ${result.code}`;
             imports.push({
               name: typeName,
               from: typegenTemplate.dst,
-              typeFrom: "./sanity-typegen",
+              typeFrom: "./types/sanity-typegen",
               type: true
             });
           }
